src/main/resources/static/css/style.css

```
@@ -0,0 +1,27 @@
+
+:root {
+    --state-blue: #1a2b49;
+    --state-gold: #b39359;
+    --state-gray: #f4f4f4;
+}
+body {
+    font-family: 'Georgia', serif;
```

Stylelint (17.6.0)

[error] Unexpected quotes around "Georgia" (font-family-name-quotes)

(font-family-name-quotes)